assets/queries/k8s/container_security_context/query.rego

```diff
@@ -0,0 +1,52 @@
+package Cx
+
+CxPolicy [ result ] {
+    document := input.document[i]
+    specInfo := getSpecInfo(document)
```

Comment:
Add metadata information to get a more specific searchKey
metadata := document.metadata